cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIHDFSSink.java

```diff
@@ -68,7 +68,7 @@
     /**
```

Comment:
CNR entry is missing.

In addition, should be some modification piece to be modified due to the new functionality? Maybe not... just checking

Reply:
Fixed in [d116da2](https://github.com/telefonicaid/fiware-cygnus/commit/d116da284c8d0d6766cb4cb5d8fa0caaefc94940)